include/boost/math/quadrature/gauss_kronrod.hpp

```diff
@@ -226,7 +161,7 @@ class gauss_kronrod_detail<T, 15, 4>
 public:
    static  std::array<T, 8> const & abscissa()
    {
-      static  std::array<T, 8> data = {
+      static  std::array<T, 8> data = { // LCOV_EXCL_START
```

Comment:
Is `LCOV_EXCL_START` documented somewhere?

Reply:
It's used by the lcov tool when collecting code coverage data (ie it's documented by the lcov tool).  For some reason GCC has trouble instrumenting arrays which are dynamically initialized, with only the final value being marked as covered, so we have to exclude those lines from coverage analysis.